src/main/java/ch/wisv/areafiftylan/products/service/OrderServiceImpl.java

```diff
@@ -206,12 +206,16 @@ public Order removeTicketFromOrder(Long orderId, Long ticketId) {
     @Override
     public String requestPayment(Long orderId) {
         Order order = getOrderById(orderId);
-        if (order.getAmount() == 0) {
+        if (order.getAmountTickets() == 0) {
             throw new IllegalStateException("Order can not be empty");
         }
         if (order.getStatus() != OrderStatus.ASSIGNED) {
             throw new UnassignedOrderException(order.getId());
         }
+        if (order.getAmount() == 0) {
+            order.setStatus(OrderStatus.PAID);
+            return "https://areafiftylan.nl/order-check?order=" + orderId;
```

Comment:
Please build this url with a String from a property. See https://github.com/AreaFiftyLAN/lancie-api/blob/master/src/main/java/ch/wisv/areafiftylan/products/service/MolliePaymentService.java#L45-L49 for an example. Maybe you can reuse a property, or else add a new one

Reply:
Fixed